docs/docker.md

```diff
@@ -0,0 +1,10 @@
+# Docker Commands
+
+## Simple commands
+```
+docker build -t ollama-image -f Dockerfile.cuda .
+```
+
+```
+docker run --name ollama-container --gpus all -v /home/user/datadir/ollama:/home/ollama -p 11434:11434 ollama-image
```

Comment:
This is a good start however it's missing the critical step that mentions installing the nvidia-container-toolkit. It's also worth noting this only works on Windows or Linux. On MacOS, docker run will exit with error `could not select device driver "" with capabilities: [[gpu]]` since there's currently no gpu extension for it.